Source/KerbalConstructionTime/Presets/PresetManager.cs

```diff
@@ -300,7 +300,7 @@ public class KCT_Preset_General : ConfigNodeStorage
     public class KCT_Preset_Time : ConfigNodeStorage
     {
         [Persistent]
-        public double OverallMultiplier = 1.0, BuildEffect = 1.0, InventoryEffect = 100.0, ReconditioningEffect = 1728, MaxReconditioning = 345600, RolloutReconSplit = 0.25;
+        public double OverallMultiplier = 1.0, BuildEffect = 1.0, InventoryEffect = 100.0, ReconditioningEffect = 1728, MaxReconditioning = 345600, RolloutReconSplit = 0.25, MergingTimePercent = 5;
```

Comment:
Other formulas represent percentages as decimals

Reply:
Also changed MergingTimePercent to MergingTimePenalty and updated GUI text.